mako_files/ucoms_decode.cc.mako

```diff
@@ -16,7 +16,7 @@ char uComsDecode::Increment(int* index, const char* input) {
 int uComsDecode::ParseValue(int index, const char* input, const char end_char) {
```

Comment:
Maybe should rename this since it's not just parsing value, it's parsing the substring.